src/com/example/module03/PassportFormCreator_Task04.java

```diff
@@ -0,0 +1,18 @@
+package com.example.module03;
+
+import java.util.Scanner;
+
+public class PassportFormCreator_Task04 {
+    public void createForms() {
+        Scanner scanner = new Scanner(System.in);
+        String name = scanner.nextLine();
+        String surname = scanner.nextLine();
+        int years = scanner.nextInt();
+        years *= 10;
```

Comment:
please introduce a new varibable for increasing the program reabability and understading for it do without reading documantation. 
so differentiate earth years and moon years in different variables.